cockpit/cockpit-sample-plugin/README.md

```diff
@@ -3,7 +3,7 @@ Sample Plugin for Camunda Cockpit
 
 This is a simple plugin that showcases the plugin system of Cockpit, the process monitoring tool of [Camunda BPM](http://docs.camunda.org).
 
-Built and tested against Camunda BPM version `7.13.0`.
+Built and tested against Camunda BPM version `7.14.0-alpha2`.
 
 ![Screenshot](screenshot.png)
 
```

Comment:
(I can't add comments to the 'Integrate into Camunda webapp' part directly.)

Depending on how you build the Webapp, this will not work as in the dev setup, we use another server to serve Cockpit. I can't speak for the result if we build a distro, but that would require another repo not mentioned in this How-To.

Should we rewrite it to deploy it on a Tomcat server/integrate it in a spring boot pom?

Reply:
The meaning of this paragraph is not very clear to me. It does not explicitly say that the dev setup is meant. When adding the dependency and running `mvn clean install` a WAR file is built, which could, in theory, be deployed to an application server. However, there is no further explanation. 
Since at a later point in the README, it is already described how to deploy the plugin to the Tomcat distribution, let's simply remove the weird paragraph with unclear meaning.